releasetasks/templates/final_verify.yml.tmpl

```diff
@@ -0,0 +1,49 @@
+{% set buildername = "{}_final_verify".format(branch) %}
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    reruns: 5
+    task:
+        provisionerId: aws-provisioner-v1
+        workerType: b2gtest
+        created: "{{ now }}"
+        deadline: "{{ now.replace(hours=6) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        scopes:
+
```

Comment:
My manual run had no scopes, so unsure what I could possibly need here, can you provide advice? https://tools.taskcluster.net/task-inspector/#COjeapHJSfCneXy6TzvqdA/